frontend/src/pages/Tasks.tsx

```diff
@@ -51,6 +96,7 @@ export default function Tasks() {
           </>
         )
       )}
+      <App />
```

Comment:
```suggestion
      <ToasterSection />
```
More suitable name than app